src/components/posts/PostsPage.js

```diff
@@ -6,37 +6,45 @@ import './style.css';
 import { Box, Stack } from '@mui/material';
 import Header from '../Header/Header';
 import dateTimetoRelativeTime from './helperfunctions';
+import { useNavigate } from "react-router-dom";
 
-const PostsPage = ({role}) => {
-  const [post, setPost] = useState({ content: '' });
-  const [postResponse, setPostResponse] = useState('');
-  const [posts, setPosts] = useState([]);
+
```

Comment:
looks like your linter locally is changing comments to double quotes... so getting lots of format changes.. go to settings in vs code and put it to single quotes

Reply:
Done